go/vt/vtctl/workflow/traffic_switcher.go

```diff
@@ -448,9 +448,10 @@ func (ts *trafficSwitcher) deleteKeyspaceRoutingRules(ctx context.Context) error
 	if err != nil {
 		return err
 	}
-	log.Infof("deleteKeyspaceRoutingRules before: %s", krr)
-	delete(krr, ts.SourceKeyspaceName())
-	log.Infof("deleteKeyspaceRoutingRules after: %s", krr)
+	for _, suffix := range tabletTypeSuffixes {
+		delete(krr, ts.SourceKeyspaceName()+suffix)
+		delete(krr, ts.TargetKeyspaceName()+suffix)
```

Comment:
Previously we were only deleting rules for the source keyspace, but now we are also doing target keyspace. Why's that?

Reply:
Good catch. Removed. I had added it earlier by mistake in a couple of places and forgot to delete this one.